profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/features/VolumeApiLiveTest.java

```diff
@@ -53,13 +55,19 @@ public void teardownTest() {
    @Test
    public void testCreateVolume() {
       assertNotNull(dataCenter);
+      
+      HashSet<String> sshKeys = new HashSet<String>();
+      sshKeys.add("hQGOEJeFL91EG3+l9TtRbWNjzhDVHeLuL3NWee6bekA=");
 
       testVolume = volumeApi().createVolume(
               Volume.Request.creatingBuilder()
               .dataCenterId(dataCenter.id())
               .name("jclouds-volume")
-              .size(3)
+              .size(4)
               .licenceType(LicenceType.LINUX)
+              .type(VolumeType.HDD)
+              .image("362c903a-df61-11e5-80a4-52540005ab80")
```

Comment:
Live tests take time. If there is a way to avoid hardcoding image ids (that eventually change when the provider updates the image list), then I'd go for that approach, even if that implies performing some additional calls. The time overhead introduced won't be that much compared to the amount of time live tests to run. WDYT?